bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -307,6 +309,14 @@ public String getLabel(Widget w) {
                             }
                         } else {
                             state = item.getState();
+                            if (stateDescription != null && stateDescription.getOptions() != null) {
```

Comment:
How are the changes within this file related to the PR? I would not have expected any changes needed in the UI layer for this.